umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md

```diff
@@ -1,7 +1,124 @@
 ---
-title: benthos-umh
-menuTitle: benthos-umh
-description:
-weight: 3000
-draft: true
+title: Benthos UMH
+menuTitle: Benthos UMH
+description: |
+  Configure OPC-UA data sources to stream data to Kafka directly in the
+  Management Console.
+weight: 2000
+draft: false
 ---
+
+[Benthos](https://www.benthos.dev/docs/about) is a stream processing tool that
+is designed to make common data engineering tasks such as transformations,
+integrations, and multiplexing easy to perform and manage. It uses declarative,
+unit-testable configuration, allowing users to easily adapt their data
+pipelines as requirements change. Benthos is able to connect to a wide range of
+sources and sinks, and can use different languages for processing and mapping
+data.
+
+[Benthos UMH](/docs/features/connectivity/benthos-umh/) is a custom extension of
+Benthos that is designed to connect to OPC-UA servers and stream data into the
+Unified Namespace.
+
+## When should I use it?
+
+OPC UA is a communication protocol coming from the OT industry, so integration
+with IT tools is necessary to stream data from an OPC UA server. With Benthos
+UMH, you can easily connect to an OPC UA server, define the nodes you want to
+stream, and send the data to the Unified Namespace.
+
+Additionally, Benthos has proven more reliable than other tools, like Node-RED,
```

Comment:
*in our tests. see also https://learn.umh.app/blog/our-open-source-docker-container-to-connect-opc-ua-with-the-unified-namespace/#testing-existing-solutions